CODEOWNERS

```diff
@@ -501,6 +503,7 @@ Makefile* @cilium/build
 /pkg/mountinfo @cilium/sig-datapath
 /pkg/mtu @cilium/sig-datapath
 /pkg/multicast @cilium/sig-datapath
+/pkg/murmur3/ @cilium/sig-agent
```

Comment:
I strongly feel this should be owned by `sig-lb` or `sig-datatpath`, since it's highly datapath (maglev) specific and only imported by that package. `pkg/maglev` is also owned by `sig-lb` after all